polkadot/node/network/availability-distribution/src/requester/mod.rs

```diff
@@ -124,6 +161,57 @@ impl Requester {
 		Ok(())
 	}
 
+	async fn request_backable_candidates_core_info<Context>(
+		&mut self,
+		ctx: &mut Context,
+		new_head: ActivatedLeaf,
+	) -> Result<Vec<(CoreIndex, CoreInfo)>> {
+		let ActivatedLeaf { hash: leaf, .. } = new_head;
+		let mut scheduled_cores = Vec::new();
+		let sender = &mut ctx.sender().clone();
+
+		let cores = recv_runtime(request_availability_cores(leaf, sender).await).await?;
+		let backable = request_backable_candidates(&cores, None, &new_head, sender).await?;
+		gum::trace!(
+			target: LOG_TARGET,
+			leaf_hash=?leaf,
+			"Got {} backable candidates", backable.len()
+		);
+
+		// now get the backed candidates corresponding to these candidate receipts
```

Comment:
```suggestion
		// now get the backed candidates corresponding to these candidate hashes
```
since the candidate receipt is part of the data we're retrieving